<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover{color:#ffffff;}QPushButton{color:#cccccc;background-color:Transparent;}</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="DraggableToolbar" name="toolbar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QPushButton" name="close_button">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>0</y>
       <width>21</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{color:#ffffff;}QPushButton{color:#cccccc;padding:0px;background-color:Transparent;}</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="min_button">
     <property name="geometry">
      <rect>
       <x>307</x>
       <y>-2</y>
       <width>21</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{color:#ffffff;}QPushButton{color:#cccccc;padding:0px;}</string>
     </property>
     <property name="text">
      <string>_</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>min_button</zorder>
    <zorder>close_button</zorder>
   </widget>
   <widget class="QLineEdit" name="user_field">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>270</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none; background-color: #ffffff;font-size:18pt; border-radius:2px; font-family:'Arial'; </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_field">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>270</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none; background-color: #ffffff; font-size:18pt; border-radius:2px; font-family:'Arial';</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_login">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>118</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: none; background-color:#98B2BD; height: 5px; color: #ffffff; font-size:16pt; font-family:'Arial'; border-radius:2px;} QPushButton:hover{background-color:#798e97;}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DraggableToolbar</class>
   <extends>QWidget</extends>
   <header>widgets.DraggableToolbar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
